### ISSUE TITLE
'ArrayIndexOutOfBoundsException: 2' for all 2-arguments methods from scala.math

### ISSUE THREAD

**Issue by vbachurin** [facaiy/math-expression-parser#7]
Hi, 

Any 2 arguments function from [scala.math](https://www.scala-lang.org/api/2.12.1/scala/math/index.html) (`max`, `min`, `pow`, `atan2`, `hypot`) throws a ArrayIndexOutOfBoundsException:

```
ex: io.github.facaiy.math.expression.Expression[String => Double,Double] = Expression(<function1>)
java.lang.ArrayIndexOutOfBoundsException: 2
at io.github.facaiy.math.expression.Expression$$anonfun$toExpression$3.apply(Expression.scala:29)
at io.github.facaiy.math.expression.Expression$$anonfun$toExpression$3.apply(Expression.scala:26)
at scala.Function1$$anonfun$compose$1.apply(Function1.scala:44)
... 38 elided
```

Code example:
```
val ex = io.github.facaiy.math.expression.MathExp.parse("pow(2, 10)")
ex.eval(Map.empty)
```

**Comment by facaiy** [comment]
Hi, will `pow(2.0, 1.0)` work for you? Thank you for the feedback :-)

**Comment by vbachurin** [comment]
@facaiy Same error. 
```
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 2
	at io.github.facaiy.math.expression.Expression$$anonfun$toExpression$3.apply(Expression.scala:29)
	at io.github.facaiy.math.expression.Expression$$anonfun$toExpression$3.apply(Expression.scala:26)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:44)
...
```
The Scala version is 2.11.12.

**Comment by facaiy** [comment]
https://github.com/facaiy/math-expression-parser/blob/775e4bf81326c445d723ba70d7028aa8b71087d1/src/main/scala/io/github/facaiy/math/expression/Expression.scala#L29

Apologies that I don't have time to fix it, I think it should be `xs(1)` here, rather than `xs(2)`. Would you like to make a contribution? Thanks.

**Comment by facaiy** [comment]
And we also need to add an unit test, like: https://github.com/facaiy/math-expression-parser/blob/775e4bf81326c445d723ba70d7028aa8b71087d1/src/test/scala/io/github/facaiy/math/expression/MathExpSuite.scala#L10

**Comment by facaiy** [comment]
Fixed by #8

**Comment by vbachurin** [comment]
@facaiy please release the fixed version.
